lib/logstasher.rb

```diff
@@ -1,8 +1,10 @@
-require 'logger'
-require 'logstash-event'
+require "logger"
+require "logstash-event"
 
 module LogStasher
   class << self
+    SILENT_LOGGER = ::Logger.new("/dev/null", level: :unknown)
```

Comment:
Should this constant be inside the `class << self``?

Being inside that means I can't refer to it normally like `LogStasher::SILENT_LOGGER `.  I would have to write `LogStasher.singleton_class::SILENT_LOGGER `